src/components/SourcesTree.js

```diff
@@ -24,21 +26,23 @@ type CreateTree = {
   sourceTree: any,
   uncollapsedTree: any,
   listItems?: any,
-  highlightItems?: any
+  highlightItems?: any,
+  debuggeeUrl: string
 };
 
 let SourcesTree = React.createClass({
   propTypes: {
     sources: ImPropTypes.map.isRequired,
     selectSource: PropTypes.func.isRequired,
     shownSource: PropTypes.string,
-    selectedSource: ImPropTypes.map
+    selectedSource: ImPropTypes.map,
+    debuggeeUrl: PropTypes.string
   },
 
   displayName: "SourcesTree",
 
   getInitialState(): CreateTree {
-    return createTree(this.props.sources);
+    return createTree(this.props.sources, this.props.debuggeeUrl || "");
```

Comment:
hmmm... I tried it just now but flow is still complaining about a potential `undefined` value